src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -845,7 +845,9 @@ export default function ApplicationForm({
             </div>
             <br />
             <div ref={jobRef}>
-              <SimpleForm simpleForm={positionInformation} />
+              <div ref={organizationFacilityRef}>
```

Comment:
why not just change the ref on the div above to `organizationFacilityRef` instead of adding a div with a new ref and still having the old `jobRef` above?

Reply:
```jobRef``` and ```organizationFacilityRef``` are for two separate fields, so I thought it made sense for them to have separate references despite pointing to a similar place on the page.